core/sbt-cloudflow/src/main/scala/cloudflow/sbt/CommonSettingsAndTasksPlugin.scala

```diff
@@ -133,7 +141,7 @@ object CommonSettingsAndTasksPlugin extends AutoPlugin {
     protocbridge.Target(
       ScalaGenerator,
       targetPath,
-      Seq("flat_package")
```

Comment:
akka grpc uses flat_package by default so I wonder why you want to change this? Can you elaborate @blublinsky ?

Reply:
the issue is that in this case it packages a lot into 1 class. as a result using generated code becomes problematic. In my case it was even using the inner class that was not defined. Typically flat_package is used to limit the amount of generated files, but it does not really help. I would, probably elevate it as a parameter, if you really want to do this

Reply:
Ok yeah can add some settings for this that the user can choose / override.